rotkehlchen/chain/evm/decoding/zerox/decoder.py

```diff
@@ -83,15 +83,11 @@ def _update_send_receive_fee_events(
             send_event.event_subtype = HistoryEventSubType.SPEND
             send_event.notes = f'Swap {send_event.amount} {send_event.asset.symbol_or_name()} via the 0x protocol'  # noqa: E501
         if receive_event is not None:
-            receive_event.counterparty = CPT_ZEROX
```

Comment:
Hmm didn't we say that you should not remove those? Why removing them from here.  It's the only place you do.